.gitignore

```diff
@@ -28,6 +28,7 @@ proguard/
 
 #IntelliJ files
 *.iml
-.idea/
+.idea/*
+!.idea/dictionaries
```

Comment:
The change to `.idea/*` is required to allow negating the rule for sub-directories.